diagnostics/physics/mld_eval.py

```diff
@@ -49,18 +59,28 @@ def plot_mld_eval(pp_root):
 
     # the two observed datasets have the same grid
     delta_deboyer = mom_rg - deboyer_mld_winter
+    logger.info("DEBOYER_MLD_WINTER: %s",deboyer_mld_winter)
+    logger.info("DELTA_DEBOYER: %s",delta_deboyer)
+
+    # Set projection of each grid in the plot
+    # For now, sst_eval.py will only support a projection for the arctic and a projection for all other domains
+    if config['projection_grid'] == 'NorthPolarStereo':
+        p = ccrs.NorthPolarStereo()
+    else:
+        p = ccrs.PlateCarree()
 
     fig = plt.figure(figsize=(8, 6))
     grid = AxesGrid(fig, 111, 
         nrows_ncols=(2, 3),
-        axes_class = (GeoAxes, dict(projection=ccrs.PlateCarree())),
+        axes_class = (GeoAxes, dict(projection = p)),
         axes_pad=0.3,
         cbar_location='bottom',
         cbar_mode='edge',
         cbar_pad=0.2,
         cbar_size='15%',
         label_mode=''
     )
+    logger.info("Successfully created grid")
 
     levels = np.arange(0, 326, 25)
     bias_levels = np.arange(-50, 51, 10)
```

Comment:
I'm fine with the hard-coded levels for now, but we might consider moving them to the YAML configuration in the future. We can let users define different levels for each variable.